kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/utils/files/MinioImpl.java

```diff
@@ -276,7 +274,7 @@ private void archiveData(Path inputFolder, String campaignName, ModeInputs modeI
         Path dataPath = modeInputs.getDataFile();
         Path newDataPath = inputFolder.resolve(ARCHIVE).resolve(getRoot(dataPath, campaignName));
 
-        if (!isDirectory(dataPath.toString())) {
+        if (Boolean.FALSE.equals(isDirectory(dataPath.toString()))) {
```

Comment:
isDirectory(dataPath.toString()).equals(Boolean.FALSE) would be better

Reply:
It's to avoid NullPointerException